data/templates/syslog/rsyslog.conf.tmpl

```diff
@@ -34,9 +34,13 @@ $outchannel {{file}},{{files[file]['log-file']}},{{files[file]['max-size']}},{{f
 {%     endif %}
 {%   else %}
 {%     if hosts[host]['port'] %}
-{{hosts[host]['selectors']}} @{{host}}:{{hosts[host]['port']}}
+{{hosts[host]['selectors']}} @{% if hosts[host]['rhost_ipv6'] %}[{{host}}]{% else %}{{host}}{% endif %}:{{hosts[host]['port']}}
 {%     else %}
+{%       if hosts[host]['rhost_ipv6'] %}
```

Comment:
You actually can write: `{%       if hosts[host] | is_ipv6 %}`, could you plase change this?